README.md

```diff
@@ -43,9 +43,11 @@ Update your app's `ios/Runner/Info.plist` file to add two keys:
 * A `GADApplicationIdentifier` key with a string value of your AdMob app ID ([identified in the AdMob UI](https://support.google.com/admob/answer/7356431)).
 * A `SKAdNetworkItems` key with Google's `SKAdNetworkIdentifier` value of `cstr6suwn9.skadnetwork` and for applovin as well
 
-**Note**
-The keys for AppLovin and Google Ads **are required** in Info.plist
-If you're not using any provider, write the line as it is otherwise provide your keys
```

Comment:
@rmahmadkhan did you test the example app on both android and ios platforms?

Reply:
Yes. Working fine on both android & iOS.